api/node/cover.md

```diff
@@ -78,6 +87,62 @@ This is your main JavaScript entry point:
 
 For a complete example, see [/examples/todo/node](https://github.com/slint-ui/slint/tree/master/examples/todo/node).
 
+## Getting Started (Deno)
```

Comment:
everything is exactly the same as for nodejs. Only the deno.json.  I wonder if it wouldn't be better to merge it and have this step 2. different only.

Reply:
There’s also the command line that’s different. But you have a point:) I’ll merge this and meditate over combining the sections;)

Reply:
Ah no, it’s also .ts vs .mjs. We could demo without typescript, but that feels like a step back.